tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectUtil.java

```diff
@@ -103,6 +104,30 @@
 {
     public static final String METADATA_FOLDER = ".metadata"; //$NON-NLS-1$
 
+
+    public static void clearMarkers( IProject proj, final String sourceId, final String makerType )
```

Comment:
Add all of these marker related methods into a new file called "MarkerUtil" instead of adding everything to ProjectUtil